test/workflow_test.go

```diff
@@ -573,6 +575,48 @@ func (w *Workflows) WorkflowWithLocalActivityCtxPropagation(ctx workflow.Context
 	return result, nil
 }
 
+func (w *Workflows) NonDeterminismSimulatorWorkflow(ctx workflow.Context) ([]string, error) {
+	logger := workflow.GetLogger(ctx).Sugar()
```

Comment:
```suggestion
	logger := workflow.GetLogger(ctx)
```
Sugaring is unused, and a better habit to not use / not use unnecessarily since it's not free.

Reply:
good catch. I had this workflow on a separate go app where I was actually using sugared log functions. missed to clean up this part